feed/feed.go

```diff
@@ -281,8 +284,9 @@ func ServiceWithDefaultFeeds(pgxStore *store.PGXStore) *Service {
 		Description: "Posts by furries with #furryart. Contains a mix of SFW and NSFW content.\n\nJoin the furry feeds by following @furryli.st",
 	}, chronologicalGenerator(chronologicalGeneratorOpts{
 		generatorOpts: generatorOpts{
-			Hashtags: []string{"furryart"},
-			HasMedia: tristate.True,
+			Hashtags:           []string{"furryart"},
+			DisallowedHashtags: []string{"ai", "aiart", "aiartist", "stablediffusion", "sdxl"},
```

Comment:
Can you move this slice out into a variable so we don't need to repeat this for each feed ? And can we just apply it to all feeds ?